lib/colorls/flags.rb

```diff
@@ -41,7 +41,13 @@ def process
 
       @args = ['.'] if @args.empty?
       @args.sort!.each_with_index do |path, i|
-        next $stderr.puts "\n   Specified path '#{path}' doesn't exist.".colorize(:red) unless File.exist?(path)
+        unless File.exist?(path)
+          $stderr.puts "\n   Specified path '#{path}' doesn't exist.".colorize(:red)
+          at_exit do
+            exit 2
+          end
```

Comment:
I kind of dislike this approach since `at_exit` registers hooks each time to be executed when the application exits. This means if we have `n` files that do not exist, it will register `n` hooks with `at_exit`.

Maybe a simple return code would work better, and also would be easier to test.

Reply:
Thank you for checking my code.
I fixed my code.
I tried to set return value instead of using at_exit method.